motion_tracker/model/model.py

```diff
@@ -24,15 +24,22 @@ def make_model(img_edge_size, backend_id):
 
     ###### DEFINE FEATURIZER APPLIED TO STARTING AND ENDING IMAGES ######
     generic_img = Input(shape=img_shape)
-    layer = Convolution2D(30, 3, 3, activation='relu', border_mode='same',
-                          dim_ordering=backend_id)(generic_img)
-    layer = MaxPooling2D(pool_size=(3, 3), dim_ordering=backend_id)(layer)
-    layer = Convolution2D(30, 3, 3, activation='relu', border_mode='same',
+    layer = Convolution2D(96, 11, 11, subsample=(4, 4), activation='relu',
+                          border_mode='same', dim_ordering=backend_id)(generic_img)
+    layer = MaxPooling2D(pool_size=(3, 3), strides=(2, 2), 
+                         dim_ordering=backend_id)(layer)
+    layer = Convolution2D(256, 5, 5, activation='relu', border_mode='same',
                           dim_ordering=backend_id)(layer)
-    layer = Convolution2D(40, 3, 3, activation='relu', border_mode='same',
+    layer = MaxPooling2D(pool_size=(3, 3), strides=(2, 2), 
+                         dim_ordering=backend_id)(layer)
+    layer = Convolution2D(384, 3, 3, activation='relu', border_mode='same',
                           dim_ordering=backend_id)(layer)
-    layer = Convolution2D(40, 3, 3, activation='relu', border_mode='same',
+    layer = Convolution2D(384, 3, 3, activation='relu', border_mode='same',
                           dim_ordering=backend_id)(layer)
+    layer = Convolution2D(256, 3, 3, activation='relu', border_mode='same',
+                          dim_ordering=backend_id)(layer)
+    layer = MaxPooling2D(pool_size=(3, 3), strides=(2, 2), 
+                         dim_ordering=backend_id)(layer)
```

Comment:
Also, not 100% sure this will run without causing an error on dimensions yet. Waiting for a different model to run and working on figuring out how to run on multiple GPU's.